server/startup/serverRunning.js

```diff
@@ -8,15 +8,12 @@ import path from 'path';
 import semver from 'semver';
 
 Meteor.startup(function() {
-	let oplogState = 'Disabled';
+	let isOplogEnabled = false;
```

Comment:
Why not `const` receiving the result of `mongo._oplogHandle && mongo._oplogHandle.onOplogEntry` directly, without the `if` statement?